nextcloudappstore/core/models.py

```diff
@@ -196,6 +202,58 @@ def _latest(self, releases):
             return None
 
 
+class AppRating(TranslatableModel):
+    app = ForeignKey('App', related_name='app', verbose_name=_('App'),
```

Comment:
I believe `related_name` should be something like `app_rating`. Same on line 209.